compose/common.yml

```diff
@@ -262,6 +287,20 @@ services:
     ports:
       - ${STAKE_POOL_API_PORT:-4010}:3000
 
+  asset-provider-server:
```

Comment:
The `handle-provider-server` in _local-testnet_ (`packages/e2e/docker-compose.yml`) has following configuration:
```
    TOKEN_METADATA_SERVER_URL: stub://
```

I'm not sure about it, but I guess `asset-provider-server` requires the same setting.
Could you please confirm @mkazlauskas ?

Reply:
Yea.. I was looking at that this afternoon, I think the `asset-projector` also needs configuration in that file.

Reply:
Asset **projector** shouldn't need it, but **provider** does

Reply:
It should be addressed here: 91f41e5fffc8a29cddbb82bf555f97492aa0e51b